spring-batch-docs/modules/ROOT/pages/step/controlling-flow.adoc

```diff
@@ -294,14 +294,20 @@ the condition of the execution having skipped records, as the following example
 
 [source, java]
 ----
-public class SkipCheckingListener extends StepExecutionListenerSupport {
+public class SkipCheckingListener implements StepExecutionListener {
+
+    @Override
+    public void beforeStep(StepExecution stepExecution) {
+        // Do something before step execution
+    }
```

Comment:
I think this can be omitted since it is the same as the default method in the interface. Do you agree?

Reply:
Sure @fmbenhassine, i agree.
I'll update the PR soon.

Reply:
@fmbenhassine  I have updated the PR.